st2actions/st2actions/notifier/notifier.py

```diff
@@ -74,29 +75,24 @@ def __init__(self, connection, queues, trigger_dispatcher=None):
             pack=ACTION_TRIGGER_TYPE['pack'],
             name=ACTION_TRIGGER_TYPE['name'])
 
-    def process(self, liveaction):
-        live_action_id = str(liveaction.id)
-        extra = {'live_action_db': liveaction}
-        LOG.debug('Processing liveaction %s', live_action_id, extra=extra)
+    def process(self, execution):
```

Comment:
While making changes, we could be more explicit and consistent and call argument `execution_db`.